src/Lykke.Backoffice.Common/CheckBrowserMiddleware.cs

```diff
@@ -53,22 +53,26 @@ public CheckBrowserMiddleware(RequestDelegate next, IEnumerable<Browser> browser
             var path = context?.Request?.Path.Value;
             if (!string.IsNullOrEmpty(path) && _skipUrls.Contains(path.ToLower()))
             {
-                
                 context.Response.StatusCode = 200;
                 await _next(context);
+                return;
             }
 
-            var useragentHeader = context.Request.Headers["User-Agent"];
+            var useragentHeader = context?.Request?.Headers["User-Agent"];
             var useragent = new UserAgent(useragentHeader);
             var supportedBrowser = CheckBrowserMajorVersion(useragent.Browser.Name, useragent.Browser.Major);
             context.Response.StatusCode = 200;
             if (!supportedBrowser)
             {
                 context.Response.StatusCode = 403;
+
+                context.Response.Headers.Add("User-Agent", useragentHeader.GetValueOrDefault());
```

Comment:
lets return received user agent in response body, not by replacing user-agent.
It is very strange if server replies with user's user-agent)